assignments/callbacks.js

```diff
@@ -24,19 +25,46 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr);
 }
+      getLength(items,arrLength);
+      function arrLength(arr) {
+        console.log(arr.length);
+      } 
+
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+
```

Comment:
So this function would be your callback function. You seem to be adding your cb into your test functions. Let's talk about this.